### ISSUE TITLE
symbol lookup error

### ISSUE THREAD

**Issue by igormalyk** [spolu/breach_core#88]
Downloaded breach install from http://bit.ly/1trVzzh (given at button click at http://breach.cc/)
`cd path/to/breach`
`./breach`

```
/path/to/breach/breach-v0.3.20-alpha.5-linux-x64/__AUTO_UPDATE_BUNDLE__/exo_browser/exo_browser: symbol lookup error: /path/to/breach/breach-v0.3.20-alpha.5-linux-x64/__AUTO_UPDATE_BUNDLE__/exo_browser/exo_browser: undefined symbol: g_type_class_adjust_private_offset
```

Chrome and chromium run ok.
x86_64 Fedora 19

**Comment by alagunambi** [comment]
Getting same issue

**Comment by argshook** [comment]
Despite other dependency issues I get, this is also where I get stuck. issue #92

**Comment by llccs** [comment]
Same error.  Gentoo 2.2, gcc version 4.6.3.

**Comment by spolu** [comment]
Hi guys,
I've made a build with Ubuntu 12.04 of the exobrowser. Can you try running it and tell me it that solves these problems: https://github.com/breach/releases/tree/master/exo_browser/v0.6.1916

**Comment by spolu** [comment]
This should be solved by the release of v0.3.22-alpha.6
Linux x64: http://bit.ly/1p8Jkmc

Make sure to check README in tarball or https://github.com/breach/breach_core/wiki/Running-Breach-on-Linux
